frontend/src/views/WorkTable/WorkTable.tsx

```diff
@@ -21,6 +23,8 @@ import { BackToTopButton } from './BackToTopButton';
 import { YearPicker } from './Filters/YearPicker';
 import { ModifiedFields } from './diff';
 
+dayjs.extend(arraySupport);
```

Comment:
[nit] This Dayjs plugin should probably rather be added in some top-level piece of code, like `src/index.ts`?

Reply:
Moved the extending to a `dayjs.ts` file in the src folder root. This can be changed later on if a more elegant way can be used..